src/routes/transactions/mappers/common/transaction-info.mapper.ts

```diff
@@ -180,13 +185,73 @@ export class MultisigTransactionInfoMapper {
     );
   }
 
-  private isCoWSwapOrder(
-    transaction: MultisigTransaction | ModuleTransaction,
-  ): boolean {
+  private isSwapOrder(transaction: { data?: `0x${string}` }): boolean {
```

Comment:
Very nitpicky: I'm under the impression that `mapSwapOrder` determines whether a transaction is a swap order or not. So this function name is a bit confusing at first glance, I'd say this one checks the data within the transaction, what about renaming to `isSwapOrderData`? Do you think it would make the class API clearer? 

Not strong opinion, I left this up to you.

Reply:
> Very nitpicky: I'm under the impression that `mapSwapOrder` determines whether a transaction is a swap order or not. So this function name is a bit confusing at first glance, I'd say this one checks the data within the transaction, what about renaming it to `isSwapOrderData`? Do you think it would make the class API clearer?
> 
> Not a strong opinion btw, I left this up to you 🙂

The goal of `mapSwapOrder` is to map a transaction to a Swap Order, it's not really to check if the transaction is a Swap Order although it can be used for that (nullability test).

The `isSwapOrder` is a utility function which works on the "transaction" level as in, it should work with any object that is a transaction-like (that's why the requirement of the argument is to have an optional `data` argument).

So in summary:
- `isSwapOrder` – returns `true` if the transaction is a Swap Order.
- `mapSwapOrder` – returns the `SwapOrder` if the transaction was successfully mapped to a Swap Order, `null` otherwise.